content/chainguard/libraries/python/build-configuration.md

```diff
@@ -37,75 +38,112 @@ The following steps allow you to determine the URL and authentication details fo
+   authentication methods for different tools.
+
+<a id="artifactory"></a>
 
-<a name="artifactory"></a>
 ### JFrog Artifactory
 
 The following steps allow you to determine the identity token and URL for accessing your organization's JFrog Artifactory repository manager.
 
 1. Select **Administration** in the top navigation bar.
 1. Select **Repositories** in the left hand navigation.
 1. Select the **Virtual** tab in the repositories view.
-1. Locate the *chainguard-python** repository row and select the elipsis (**...**) in the last column on the right.
+1. Locate the *chainguard-python** repository row and select the ellipsis (**...**) in the last column on the right.
 1. Select **Set Me Up** in the dialog.
 1. Select **Generate Token & Create Instructions**
 1. Copy the generated token value to use as the password for authentication.
 1. Select **Generate Settings**.
-1. Copy the value from one of the *URL* fields. The are all identical. For example, `https://exampleorg.jfrog.io/artifactory/chainguard-python` with `exampleorg`. 
+1. Copy the value from one of the *URL* fields. The are all identical. For
+   example, `https://exampleorg.jfrog.io/artifactory/chainguard-python` with
+   `exampleorg`. Note that for use
+   with build tools you must append `simple` to the URL so that the package
+   index is used successfully -
+   `https://exampleorg.jfrog.io/artifactory/chainguard-python/simple/`.
 
-<a name="nexus"></a>
-## Sonatype Nexus Repository
+<a id="nexus"></a>
 
-The following steps allow you to determine the URL and authentication details for accessing your organization's Sonatype Nexus repository group.
+### Sonatype Nexus Repository
 
-1. Click **Browse** in the **Welcome** view or the browse icon (cube) in the top navigation bar.
-1. Locate the **URL** column for the *chainguard-python* repository group and press **copy**. The URL should take the following format: `https://repo.example.com/repository/chainguard-python/` .
-1. Use your configured username and password unless **Security** - **Anonymous Access** - **Access** - **Allow anonymous users to access the server** is activated. Details vary based on your configured authentication system.
+The following steps allow you to determine the URL and authentication details
+for accessing your organization's Sonatype Nexus repository group.
 
-## Configuring Build Tools
+1. Click **Browse** in the **Welcome** view or the browse icon (cube) in the top navigation bar.
+1. Locate the **URL** column for the *chainguard-python* repository group and
+   press **copy**. The URL should take the following format:
+   `https://repo.example.com/repository/chainguard-python/`. Note that for use
+   with build tools you must append `simple` to the URL so that the package
+   index is used successfully -
+   `https://repo.example.com/repository/chainguard-python/simple/`.
+1. Use your configured username and password unless **Security** - **Anonymous
```

Comment:
Can you just say the last piece (**Allow anonymous users to access the server**) here? It is a little hard to parse

Reply:
I can turn it around but essentially its a company policy thing and varies...